src/components/templates/PartyMap/components/Map/Map.tsx

```diff
@@ -214,14 +214,16 @@ export const Map: React.FC<MapProps> = ({
 
   const roomOverlay = useMemo(
     () =>
-      venue.rooms?.map((room) => (
-        <MapRoom
-          key={room.title}
-          venue={venue}
-          room={room}
-          selectRoom={() => selectRoom(room)}
-        />
-      )),
+      venue.rooms?.map((room) =>
```

Comment:
I think a better approach would be to filter them first, based on `isEnabled` prop and then to map without a ternary operator

Reply:
Usually I'd be against this extra iteration, but this array is small and has no impact on the performance.

Reply:
> Usually I'd be against this extra iteration, but this array is small and has no impact on the performance. +1